src/main/java/org/recordrobotics/charger/control/IControlInput.java

```diff
@@ -18,4 +18,11 @@ public interface IControlInput {
 	 * @return [-1, 0) - left; (0, 1] - right
 	 */
 	double getDriveLat();
+
+	/**
+	 * Claw state input (cone & cube) value
+	 *
+	 * @return [-1, 0) - cone; 0 - neutral; (0, 1] - cube 
```

Comment:
This isn't right either. First of all the function outputs discrete values, so intervals don't make sense.
Second, this should be an enum for clarity

Reply:
how would you write a enum

Reply:
See 2022_Rewrite IControlInput for reference

Reply:
done

Reply:
Not done, you defined the enum, but the logic does not use it yet.